requirements_liveonly.txt

```diff
@@ -0,0 +1,11 @@
+websockets==9.1
+pandas==1.3.0
+hjson==3.0.2
+numpy>=1.20.1
```

Comment:
Is there a specific reason numpy is not fixed?

Reply:
Nope, just copied over from the original requirement file I think. I'll fix that, thank you